File: modules/people.py
class People:

    def __init__(self):
        self.People = []

    def addPerson(self, person):
        self.People.append(person)

    def listPeople(self):
        for person in self.People:
            print("Name: " + person.firstName + " " + person.surname)
            print("Age: " + str(person.age))

File: javascript/people.js
class People {
  constructor() {
    this._people = [];
  }

  get People() {
    return this._people;
  }

  listPeople() {
    for (var i = 0; i < this.People.length; i++) {
      console.log(
        "Name: " + this.People[i].FirstName + " " + this.People[i].Surname
      );
      console.log("Age: " + this.People[i].Age);
    }
  }

  addPerson(person) {
    this.People.push(person);
  }
}

module.exports = People;
